src/actorCounter.js

```diff
@@ -0,0 +1,6 @@
+function actorCounter(list) {
+  const counter = document.getElementById('actor-count');
+  counter.innerHTML = `Actor Count (${list.splice(-27).length})`;
+}
+
+module.exports = actorCounter;
```

Comment:
- [ ] Still, functions in the actorCounter JS logic is using the `function` keyword instead of the `arrow` function of the `es6` version of JS please update them.